en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the release policy for Silverstripe CMS. It applies to all [commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the silverstripe GitHub organisation that are not commercially supported are updated on a best effort basis.
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+The purpose of this policy is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+## Release types and cadence
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its release philosophy.
+
+### Major releases `(x.0.0)`
```

Comment:
The following major release policy information is missing:

- At any given time, only two major release lines are officially supported.
- symfony support commitments
- context from the "Active development" section (it looks like you might be changing the context of the policy from talking about a major release *line* to just the major x.0.0 release itself? Whichever of those is intended needs to be very explicitly stated)
- At any given time, only two major release lines are officially supported.


Other things are also missing but are explicitly pointed out in other comments.

Reply:
> At any given time, only two major release lines are officially supported.

Have added sentence to "Support lifecycle" section

> symfony support commitments

Added as sub-heading under "Dependency management" 

> Active development

This was a very confusing concept that previously applied to both the next-minor and the current major and sometimes the next major (I think??)

The old major policy had this "Only one major release line is in active development at any given time." - which is just wrong?

I've simply remove references to "Active development", I don't think it's a helpful concept

On the new support-timeline it's been changed to "In development" and refers exclusively to up-coming minor releases, not majors